code/modules/hydroponics/eggincubator.dm

```diff
@@ -0,0 +1,108 @@
+/obj/machinery/egg_incubator
+	name = "egg incubator"
+	icon = 'icons/obj/virology.dmi'
+	icon_state = "incubator"
+	layer = 2.9

```

Comment:
The microwave doing it was shit and oldcode, it shouldn't have ever been like that ever.